deploy/kubernetes/base/node_setup/windows/node.yaml

```diff
@@ -0,0 +1,82 @@
+#TODO(#40): Force DaemonSet to not run on master.
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-node-win
+spec:
+  selector:
+    matchLabels:
+      app: gcp-compute-persistent-disk-csi-driver-win
+  template:
+    metadata:
+      labels:
+        app: gcp-compute-persistent-disk-csi-driver-win
+    spec:
+      # Host network must be used for interaction with Workload Identity in GKE
+      # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
+      # this requirement when issue is resolved and before any exposure of
+      # metrics ports. But hostNetwork is not working for Windows, might be an issue
+      # when deploying on GKE Windows node.
+      # hostNetwork: true
+      priorityClassName: csi-gce-pd-node
+      serviceAccountName: csi-gce-pd-node-sa
+      nodeSelector:
+        kubernetes.io/os: windows
+      containers:
+        - name: csi-driver-registrar
+          image: gcr.io/k8s-staging-csi/csi-node-driver-registrar:amd64-windows-v20200428-v1.3.0-26-g510710d5
```

Comment:
These images should be substituted in the alpha overlay

Reply:
we don't have another image yet, I think. Currently I see this is in staging. We need to check when it is able to release.

Reply:
Yes, but I think we should follow the pattern of the linux node.yaml and not specify a tag in the base, and instead override the tag in the kustomization.yaml